trunkclub/create-react-app
# V11 v2

#### What's this do?

Pulls the latest upstream changes from create-react-app (v2.1)

#### Why are we doing this? (w/ JIRA link if applicable)

@trunkclub/build hasn't been updated in awhile and create-react-app v2.1 provides some nice new features

#### Who should review this PR?

@trunkclub/ui

#### Anything the reviewer(s) should know to review effectively?

Yes, please see https://github.com/trunkclub/tcweb-build/blob/1c0ff3c2afc2c6725e86f7ad4c9c16355c29703f/docs/UPGRADING-v11.md for a list of breaking changes.  If you disagree with any, please reach out.

#### How should this be tested?

An alpha release (@trunkclub/build@11.0.10) is currently used in production for `pos_app`.  I'd like a few other teams to start using these versions as well to see if I missed anything.

#### Dependencies for merging? Releasing to production?

Nope
